docker/helm/templates/sts-coordinator.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: coordinator-server
+  labels:
+  {{- include "fluss.labels" . | nindent 4 }}
+spec:
+  serviceName: coordinator-server-hs
+  replicas: 1 # can only be 1 for now
+  selector:
+    matchLabels:
+      {{- include "fluss.selectorLabels" . | nindent 6 }}
+      app.kubernetes.io/component: coordinator
+  template:
+    metadata:
+      labels:
+        {{- include "fluss.selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/component: coordinator
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}-coordinator
+          image: "{{.Values.image.repository}}:{{ .Values.image.tag }}"
```

Comment:
Do we need to add `{{.Values.image.registry}}`, and change this to `{{.Values.image.registry}}/{{.Values.image.repository}}:{{ .Values.image.tag }}`